auth.go

```diff
@@ -86,7 +87,7 @@ the identity that performed the request, when this token expires, and
 the permit belonging to this identity.
 */
 type Token struct {
-	Identity string
+	UserId UserId
```

Comment:
This code has not been formatted with gofmt. You MUST use an editor that automatically runs gofmt on every save, otherwise you're just wasting your time (and others).